.git-blame-ignore-revs

```diff
@@ -1,7 +1,20 @@
 # This file is read by GitHub and `git blame --ignore-revs-file`.
 # See: https://docs.github.com/en/repositories/working-with-files/using-files/viewing-a-file#ignore-commits-in-the-blame-view
 
-# Revert "Change brace space in clang-format (#2627)"
+# --------------------------------------
+
+# The following section was generated by running: `git log --format="# %s%n%H" .clang-format`
```

Comment:
I wrote a comment to relatively easily update this file.

Unfortunately, there are commits which change both styling and something else, so I had to go through them and exclude them manually.